src/Artifacts.jl

```diff
@@ -357,7 +358,7 @@ function download_artifact(
             # Since tree hash calculation is still broken on some systems, e.g. Pkg.jl#1860,
             # and Pkg.jl#2317 so we allow setting JULIA_PKG_IGNORE_HASHES=1 to ignore the
             # error and move the artifact to the expected location and return true
-            ignore_hash = get(ENV, "JULIA_PKG_IGNORE_HASHES", nothing) == "1"
+            ignore_hash = bool_env("JULIA_PKG_IGNORE_HASHES")
```

Comment:
Perhaps
```suggestion
            ignore_hash = get_bool_env("JULIA_PKG_IGNORE_HASHES")
```

Reply:
Not sure what you mean, is this a suggestion to rename the function?

Reply:
Yeah. Just a minor suggestion